modules/ROOT/pages/to-change-raml-version.adoc

```diff
@@ -24,36 +24,29 @@ NOTE: New asset versions count toward your organization's xref:to-create-an-asse
 
-image::ex-publish-a-new-asset.png[]
 
 Only change the API version when an updated API specification has changes that are not backward compatible from the new version to the old version.
 
 == Change RAML or OAS Version
 
-For RAML and OAS API specifications within Exchange, you can change the version number from within Exchange using the *Add new version* button in the Exchange user interface.
+For RAML and OAS API specifications within Exchange, change the version number from within Exchange using the *Add version* button. 
 
-image::ex-add-new-version.png[]
+. From the asset details pane, click *Add version*. 
+. (Required) Select a *Method* for the API specification.
+. (Required) From *File upload* click *Choose file*. For OAS, choose an OAS file with the .yaml or .json file types. For RAML, choose a RAML file with the .raml file type. Zip files are also accepted with the file to upload residing in the zip file’s root directory.
+. Specify the *Main file* if the uploaded file is a zip. Otherwise, Exchange uses the same file name as the file you upload. 
+. (Required) In *Version* , enter a new version for the asset.
+. (Required) In *the *API version*, enter the version of the API specification.
+The Version and API version values must agree. You can only choose minor version updates for a given API version. To set a major Version number, also increase the API version. If you choose minor or patch version updates, the API versions should not changed.
 
-This opens the *Publish a new asset* window (as shown in <<deploy-a-new-api-asset>>).
+. (Optional) Select a *Lifecycle state*. Default is *Stable*.
+. Click *Publish*
 
-These fields are required: *File upload*, *Main file*, and *Version*.
+You can also change the version using the *Manage versions* button. See xref:manage-versions.adoc[Manage Versions] for more information.
 
-[%header%autowidth.spread]
-|===
-|Field |Description
-|File upload |For OAS, choose an OAS file with the .yaml or .json file types.
-For RAML, choose a RAML file with the .raml file type. Zip files are also accepted with the 
-file to upload residing in the zip file's root directory.
-|Main file |Specify the main file if the uploaded file is a zip. Otherwise, Exchange uses the same file name as the file you upload.
-|Version |The new version number. The Version and API version values need to agree 
-in that you can only choose minor version updates for a given API version. To set a major
-Version number, also increase the API version. If you choose minor or patch version 
-updates, you should not change the API version.
-|===
-
-You can also change the version number for RAMLs as described in the next sections.
+Change the version number for RAMLs as described in the next sections.
```

Comment:
Going forward, do not refer to other text units except by name, to ensure reusability.